products/opensuse/main.pm

```diff
@@ -195,9 +195,7 @@ sub load_boot_tests {
         set_var("DELAYED_START", "1");
         loadtest "autoyast/pxe_boot";
     }
-    elsif (check_var('ARCH', 's390x')) {
-        loadtest "installation/bootloader_s390";
-    }
+    elsif (load_bootloader_s390x()) { }
```

Comment:
I'm not sure if I like it. I guess we can move it up or down and call unconditionally. e.g. 
```loadtest "installation/bootloader" unless load_bootloader_s390x()```

I also don't see a problem having check everywhere, as we anyway run check_var on ARCH, so it's not really any improvement.

Reply:
Agree, that moving it down looks better, I'll change it.

The intention of the patch was DRY.